app/jobs/close_patients_job.rb

```diff
@@ -30,8 +30,14 @@ def perform(*_args)
         patient[:monitoring_reason] = 'Completed Monitoring (system)'
       end
 
-      # Send closed email to patient if they are a reporter
-      PatientMailer.closed_email(patient).deliver_later if patient.save! && patient.email.present? && patient.self_reporter_or_proxy?
+      # Send closed email or SMS to patient if they are a reporter
+      if patient.save! && patient.self_reporter_or_proxy?
```

Comment:
Do you have a `else` case in mind for the second `if` statement? Does the system need to inform that a close message couldn't be sent because the patient had no usable contact method? Can you use the `has_usable_contact_method?` scope here? Then the internal `if` statement becomes `if else` instead of `if elsif`.

Reply:
The `has_usable_preferred_contact_method` scope checks only the `preferred_contact_method` on the patient. The way this was written before this PR was slightly different in that if there was an email present, then it would send the closed email regardless of if it was the preferred method of contact or not. I was preserving that logic, but if that should change now then I can do that. IF we do change the logic then we also need to consider what should happen if the preferred contact method is a phone call.

@holmesie do you have thoughts on this?

Reply:
We don't need to handle phone call (at least for this PR), and I do think now this needs to use the preferred contact method (or do nothing).

Reply:
@holmesie @Bialogs Can we expect the case of the preferred contact method to always be the same?

As in always `'E-mailed Web Link'` and never `'e-mailed web link'`

Reply:
Yes

Reply:
This should be addressed now